pyproject.toml

```diff
@@ -59,7 +59,6 @@ kafka-python = "*"
 logstash-formatter = "*"
 python-dateutil = "*"
 pytz = "*"
-uvicorn = "^0.29.0"
```

Comment:
Maybe for the sake of consistency we should add `uvicorn = "*"` like in drift-backend and system-baselines

Reply:
oh, I forgot that, and I wanted to change that... thank you